boards/same54-xpro/include/board.h

```diff
@@ -75,6 +75,14 @@ extern "C" {
 #define BTN0_MODE           GPIO_IN_PU
 /** @} */
 
+/**
+ * @name ATA6561 STANDBY pin definition
+ * @{
+ */
+#define AT6561_STBY_PIN            GPIO_PIN(PC, 13)
```

Comment:
Maybe this could have a name like CAN_TRANSCEIVER_STBY..

Reply:
AT6561 is the CAN transceiver name. I followed the naming of the other components defined in the board

Reply:
Sure but the name is not relevant - you want to use a generic name so the test app where you are using it can be generic too.

On a second note, shouldn't that pin be handled by the driver in the first place?
Only the CAN driver would know when to put the transceiver into standby mode.

Reply:
It probably is the application that knows best how to handle the standby pin if it exists (or at least it is up to the board) the standby pin is often usable to stay silent on the bus but still receive

Reply:
But we have `CANOPT_STATE_SLEEP` for that. When set, the driver should put the transceiver into standby mode - not the application.

Reply:
https://e2e.ti.com/support/interface-group/interface/f/interface-forum/755985/tcan1042-q1-difference-between-silent-mode-and-standby-mode

seems like for that transceiver standby isn't as transparent (listen only would not work with that, for other transceivers it might, more like silent by TI definition)

maybe the `can_trx` driver interface should be used for that (driver/tja1042 seems to have a standby pin)